connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaPropertyTransformer.java

```diff
@@ -27,15 +22,7 @@ public class KafkaPropertyTransformer {
 
   private static final Logger LOG = LoggerFactory.getLogger(KafkaPropertyTransformer.class);
 
-  private static final ObjectMapper objectMapper =
-      new ObjectMapper()
-          .registerModule(new Jdk8Module())
-          .registerModule(DefaultScalaModule$.MODULE$)
-          .registerModule(new JavaTimeModule())
-          // deserialize unknown types as empty objects
-          .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
-          .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-          .enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES);
```

Comment:
This feature looks quite specific (haven't seen it anywhere else) - maybe we can keep it?